frontend/src/components/form/base/BaseComponent.vue

```diff
@@ -28,10 +29,11 @@
 import { ValidationProvider } from 'vee-validate'
 import { formComponentPropsMixin } from '@/mixins/formComponentPropsMixin.js'
 import { VTextField } from 'vuetify/lib'
+import VTiptapEditor from '@/components/form/tiptap/VTiptapEditor.vue'
 
 export default {
   name: 'BaseComponent',
-  components: { ValidationProvider, VTextField },
+  components: { VTiptapEditor, ValidationProvider, VTextField },
```

Comment:
Why this change?

Reply:
[Remove automatically added reference by PhpStorm](https://github.com/ecamp/ecamp3/pull/4376/commits/0d9b800425b80a53321d9a831c5c4f701afbb579)